### DISCUSSION TITLE
I believe infinite scroll would be better than clicking on the pagination

### DISCUSSION THREAD

**Discussion by avelino** [community/community#8595] [Other Feature Feedback, Questions, & Ideas]
when I am looking for a code it is more practical to keep scrolling down instead of clicking page and waiting for the page to reload

**Comment by tclem** [comment]
I like this idea too, good feedback.

**Comment by salolivares** [comment]
Agreed. I want my eyes to focus on what I'm scrolling past and not have to lose focus trying to navigate to another page.

**Comment by axilis-marko** [comment]
Agreed. At the least, I'd like for an option to increase the number of commits returned to the `/commits` page.
